### PR TITLE
Added FullTimeEquivalency values to StudentSchoolAssociations

### PR DESCRIPTION
**Pull request by markramonDL**
added <FullTimeEquivalency>1.00</FullTimeEquivalency> to all StudentSchoolAssociation records to support reporting use case.

### CODE REVIEW

File: sample-data/Samples/Sample XML/BearCreek-StudentEnrollment.xml [hunk 1]
```diff
@@ -2689,7 +2796,7 @@
 		<EntryDate>2021-08-23</EntryDate>
 		<EntryGradeLevel>uri://ed-fi.org/GradeLevelDescriptor#First grade</EntryGradeLevel>
 		<GraduationPlanReference ref="GPLN_Minimum-BCES"/>
-		<FullTimeEquivalency>0.75</FullTimeEquivalency>
```
Comment (blmeyers) [inline sample-data/Samples/Sample XML/BearCreek-StudentEnrollment.xml]: @vimayya Example of one changing from 0.75 to 1.00.

**Review by blmeyers — CHANGES_REQUESTED** [review]
Pending feedback from Vinaya around 0.75 value changes.

File: sample-data/Samples/Sample XML/BearCreek-StudentEnrollment.xml (hunk 1 above)
Comment (vimayya) [reply sample-data/Samples/Sample XML/BearCreek-StudentEnrollment.xml]: @blmeyers @markramonDL If there was a value before, let us not change it.

Comment (markramonDL) [reply sample-data/Samples/Sample XML/BearCreek-StudentEnrollment.xml]: restored original values.

**Review by blmeyers — APPROVED** [review]

**Pull request merged**